src/_ffab-hover-active.scss

```diff
@@ -0,0 +1,24 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Hover / Active State (ffab)
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$ffab-has-hover-active-state: true !default;
+$ffab-hover-active-transition-speed: 150ms !default;
+$ffab-hover-value: -1px !default;
+$ffab-active-value: 1px !default;
```

Comment:
It's starting 1px up so it's going 2px down (1px to get back to it's original position and one more pixel with the transform).